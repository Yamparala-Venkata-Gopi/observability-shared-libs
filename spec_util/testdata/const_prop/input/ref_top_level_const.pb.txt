# MethodTemplate proto

method_id: {
  name: "new_method_with_refs"
}

arg_templates: {
  key: "arg1"
  value: {
    ref: {
      method_index: 0
      arg_ref: {
        key: "primitive_constant_arg"
        data_ref: {
          primitive_ref: {}
        }
      }
      akita_annotations: {
        is_free: true
        is_sensitive: true
      }
    }
  }
}

arg_templates: {
  key: "arg2"
  value: {
    ref: {
      method_index: 0
      arg_ref: {
        key: "struct_constant_arg"
        data_ref: {
          struct_ref: {
            full_struct: {}
          }
        }
      }
    }
  }
}
