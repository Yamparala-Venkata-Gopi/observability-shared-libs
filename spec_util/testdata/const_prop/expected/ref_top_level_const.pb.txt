# MethodTemplate proto

method_id: {
  name: "new_method_with_refs"
}

arg_templates: {
  key: "arg1"
  value: {
    value: {
      primitive: {
        type_hint: "string_constant_1"
        string_value: {
          value: "7c73565e-0d62-49b0-84e7-3063f4d53d9c"
        }
        akita_annotations: {
          is_free: true
          is_sensitive: true
        }
      }
    }
  }
}

arg_templates: {
  key: "arg2"
  value: {
    value: {
      struct: {
        fields: {
          key: "foo"
          value: {
            primitive: {
              type_hint: "string_constant_2"
              string_value: {
                value: "1aa4e597-a29d-42d4-8134-3e4ee73aa7a1"
              }
            }
          }
        }
      }
    }
  }
}
